### PR TITLE
update webpage

### PR DESCRIPTION
**Pull request by CeresBarros**
@eliotmcintire, can you check that this can be merged? Not sure if there are incomplete things from your end.

### CODE REVIEW

File: training/R/installpkgs.R
Comment (eliotmcintire) [inline training/R/installpkgs.R:1]: This has *both* `install.packages` and `install_github` for `SpaDES.project`. That is wrong.

File: training/R/pkgList.R
Comment (eliotmcintire) [inline training/R/pkgList.R:1]: In general, I am "against" package version numbering (until you need a static version, like for a publication). It creates a laziness about code (a false sense of static R package world) and it gets annoying as Rstudio keeps prompting you for updating.

**Review by eliotmcintire — APPROVED** [review]

File: training/R/installpkgs.R
Comment (CeresBarros) [reply training/R/installpkgs.R:1]: yes, it's just a temp solution. I could've commented the `install.packages`

File: training/R/pkgList.R
Comment (CeresBarros) [reply training/R/pkgList.R:1]: Where are you seing that?

Comment (eliotmcintire) [reply training/R/pkgList.R:1]: The whole `installPkgs.R`

**Pull request merged**